### PR TITLE
fixed negative values selection bug

### PR DESCRIPTION
**Pull request by mvgaliev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-visuals-dotplot",
-  "version": "0.9.0",
+  "version": "0.9.1",
```
Comment (ignatvilesov) [inline package.json]: Please increase version of the visual to 1.0.1 due to minimal version at Office Store

File: pbiviz.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "displayName": "Dot Plot",
     "guid": "DotPlot1442374105856",
     "visualClassName": "DotPlot",
-    "version": "0.9.0",
+    "version": "0.9.1",
```
Comment (ignatvilesov) [inline pbiviz.json]: Please increase version of the visual to 1.0.1 due to minimal version at Office Store

File: src/visual.ts [hunk 3]
```diff
@@ -296,16 +296,17 @@ module powerbi.extensibility.visual {
...
             const dataPointsGroup: DotPlotDataGroup[] = [],
-                minDots = minValue / (maxValue / maxDots);
+                minDots = minValue / (maxValue / maxDots),
+                additionalValue = minValue <= 1 ? -minValue + 1 : 0; // negative values scales incorrect
 
             const dotScale: LogScale<number, number> = d3.scale.log()
-                .domain(DotPlot.getDomain(minValue, maxValue))
-                .range(DotPlot.getDomain(minDots, maxDots))
+                .domain(DotPlot.getDomain(minValue + additionalValue, maxValue + additionalValue))
+                .range(DotPlot.getDomain(minDots > maxDots ? 1 : minDots, maxDots))
```
Comment (ignatvilesov) [inline src/visual.ts]: Why minimal value is `1`? Can we use `0`?

Comment (mvgaliev) [inline src/visual.ts]: There is a minimum of 1 point for a value so we can't.

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,2 @@
+## 1.0.1
+* Fixed negative value building while selection in other visuals.
```
Comment (ignatvilesov) [inline CHANGELOG.md]: `Fixed negative value building while selection is applied to other visuals`

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,4 +1,7 @@
 ## 1.1.0
 * Updated packages
 * Added localization
-* Removed jQuery as dependencies
\ No newline at end of file
+* Removed jQuery as dependencies
+
+## 1.0.1
```
Comment (ignatvilesov) [inline CHANGELOG.md]: Why do you include `1.0.1` if we have a greater one?

File: src/visual.ts [hunk 6]
```diff
@@ -296,16 +296,17 @@ module powerbi.extensibility.visual {
                 .range([dotsTotalHeight, DotPlot.MinAmountOfDots]);
 
             const dataPointsGroup: DotPlotDataGroup[] = [],
-                minDots = minValue / (maxValue / maxDots);
+                minDots = minValue / (maxValue / maxDots),
+                additionalValue = minValue <= 1 ? -minValue + 1 : 0; // negative values scales incorrect
```
Comment (ignatvilesov) [inline src/visual.ts]: It'd be better to more `1` to a static variable

File: src/visual.ts (hunk 3 above)
Comment (ignatvilesov) [inline src/visual.ts]: It'd be better to more `1` to a static variable

**Pull request merged**